[readme] Update installation guide (#1988)
flant/werf: README.md

- Helm 3 support [#1606](https://github.com/flant/werf/issues/1606).
- Userspace builds without Docker daemon (as in kaniko) [#1618](https://github.com/flant/werf/issues/1618).

> ## Complete Feature List

Author: done

- Minimum required version is 1.9.0.
- Version 2.14.0 or newer is required to use [Git Submodules](https://git-scm.com/docs/gitsubmodules).

<!-- WERF DOCS PARTIAL END -->

## Installing werf

There are a lot of ways to install werf, but using [multiwerf](https://github.com/flant/multiwerf) is a recommended practice both for local development and CI usage. 

> The other approaches are also available in [Installation guide](https://werf.io/documentation/guides/installation.html)

<!-- WERF DOCS PARTIAL BEGIN: Installing with multiwerf -->

#### Unix shell (sh, bash, zsh)

##### Installing multiwerf

```shell
# add ~/bin into PATH
export PATH=$PATH:$HOME/bin
echo 'export PATH=$PATH:$HOME/bin' >> ~/.bashrc

# install multiwerf into ~/bin directory
mkdir -p ~/bin
cd ~/bin
curl -L https://raw.githubusercontent.com/flant/multiwerf/master/get.sh | bash
```

##### Add werf alias to the current shell session

```shell
. $(multiwerf use 1.0 stable --as-file)
```

##### CI usage tip

> To ensure that multiwerf is exist and executable use `type` command:

Reviewer: ```suggestion
To ensure that multiwerf exists and is executable, use the `type` command:
```

```shell
type multiwerf && . $(multiwerf use 1.0 stable --as-file)
```

> The command prints a message to stderr in case if multiwerf is not found. Thus, diagnostic in CI environment becomes simpler. 

Reviewer: ```suggestion
The command prints a message to stderr if multiwerf is not found. Thus, diagnostics in a CI environment becomes simpler. 
```

All changes go through all stability channels:

- `alpha` channel can bring new features but can be unstable.
  We do **not guarantee** backward compatibility between `alpha` releases.
- `beta` channel is for more broad testing of new features to catch regressions.
  We do **not guarantee** backward compatibility between `beta` releases.
- `ea` channel is mostly safe to use and can even be used in non-critical environments or for local development.
  We do **not guarantee** backward compatibility between `ea` releases.
- `stable` channel is mostly safe to use and we encourage to use this version everywhere.
  We **guarantee** that `ea` release should become `stable` not earlier than 1 week after internal tests.
  We **guarantee** backward compatibility between `stable` releases within the minor branch (1.1.x).
- `rock-solid` channel is a generally available version and recommended for use in critical environments with tight SLA.
  We **guarantee** that `stable` release should become a `rock-solid` release not earlier than 2 weeks of broad testing.
  We **guarantee** backward compatibility between `rock-solid` releases within the minor branch (1.1.x).

> The relations between channels and werf releases described in [multiwerf.json](https://github.com/flant/werf/blob/multiwerf/multiwerf.json). The using of werf by channel should be organized with [multiwerf](https://github.com/flant/multiwerf).   

Reviewer: ```suggestion
The relations between channels and werf releases are described in [multiwerf.json](https://github.com/flant/werf/blob/multiwerf/multiwerf.json). The usage of werf within the channel should be carried out with [multiwerf](https://github.com/flant/multiwerf).   
```
Здесь речь об использовании werf **в пределах** некоего канала?

> Stability channels and frequent releases allow receiving continuous feedback on new changes, quickly rolling problem changes back, ensuring the high stability of the software and saving an acceptable development speed at the same time.

Reviewer: ```suggestion
Stability channels and frequent releases allow receiving continuous feedback on new changes, quickly rolling problem changes back, ensuring the high stability of the software, and preserving an acceptable development speed at the same time.
```
